test/test-eth-trezor-keyring.js

```diff
@@ -694,4 +694,70 @@ describe('TrezorKeyring', function () {
       }
     });
   });
+
+  describe('getVendor', function () {
+    it('should call TrezorConnect.getFeatures if we dont have vendor name', async function () {
+      sinon
+        .stub(TrezorConnect, 'getFeatures')
+        .callsFake(() => Promise.resolve({}));
+
+      await keyring.getVendor();
+      assert(TrezorConnect.getFeatures.calledOnce);
+    });
+
+    it('should return null when getFeatures not success', async function () {
+      sinon
+        .stub(TrezorConnect, 'getFeatures')
+        .callsFake(() => Promise.resolve({}));
+
+      const vendor = await keyring.getVendor();
+      assert.equal(vendor, null);
+    });
+
+    it('should return trezor when getFeatures does not have minor_version or patch_version', async function () {
```

Comment:
What do you think about adding a test for when `TrezorConnect.getFeatures` rejects with an error?

Reply:
Added.